### ISSUE TITLE
[Feature] Replace the Vanilla Book

### ISSUE THREAD

**Issue by MissPotato** [shedaniel/RoughlyEnoughItems#1376]
### What is your feature request?

As we all know the Vanilla Recipe Book is subpar for modded minecraft. A lot of mods just flat out disable it because of this. I propose instead of just disabling it, making it so the recipe book acts like a toggle for the REI ui.  When you click the recipe book with this mod installed the recipe book will visually indent, and the normal REI interface opens on the side.  Wanna be clear, I'm not asking for a whole new gui, just a different way of toggle the gui off and on.

In short, disable the replace book UI and replace the button with the same toggle for the O keybind

### What do you think this change is of?

- [X] Visual
- [ ] Recipe Lookup
- [ ] Cheat Mode
- [ ] Plugin Integration / JEI Plugin Compatibility
- [ ] Others

**Comment by rozbrajaczpoziomow** [comment]
I think there should still be an option to completely disable the recipe book somewhere in the options, if this was added.

**Comment by MissPotato** [comment]
I agree, I also think there might be an issue if other mods hide the book. I think Quark for example has an option for that.

**Comment by shedaniel** [comment]
Duplicate of #919
I have considered this before and really wanted to do this as well

**Comment by MissPotato** [comment]
I have no idea how I missed this, sorry about that!
